File: Modul 4/Soal 3/PRAK303-2210817210006-Farlyhaydy H.Djalil.py
i=0
while True:
    i=i+1
    bil=int(input())
    if (bil > 0):
        print("positif\n")
    elif (bil < 0):
        print("negatif\n")
    else :
        print("nol\n")
    if (i>2):
        break

File: Modul 4/Soal 3/PRAK303-2210817210006-Farlyhaydy H.Djalil.c
#include <stdio.h>

int main()
{
   int N, i=0;
   while(i<=2){
   
   scanf("%d", &N);
   if(N>0){
   printf("positif\n\n");
   }else if(N<0){
   printf("negatif\n\n");
   }if(N==0)
   printf("nol");
   i++;
   }
}



